CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/components/ColumnVisibilityContextMenu.java

```diff
@@ -115,12 +117,20 @@ public void init() {
         });
 
         showDefaultColumnsMenu = createCustomMenu(DEFAULT_COLUMNS, e -> {
+            TableDefaultColumns tableDefaultColumnsProvider = Lookup.getDefault().lookup(TableDefaultColumns.class);
+            
+            List<Integer> ids = tableDefaultColumnsProvider.getDefaultColumns(getTableViewTopComponent().getCurrentGraph()).stream()
+                    .map(GraphAttribute::getId)
+                    .collect(Collectors.toList());
+
+            List<Tuple<String, Attribute>> filteredCols = extractColumnAttributes(table.getColumnIndex().stream()
```

Comment:
final but this could potentially also be inserted into the function call below as it was previously